### ISSUE TITLE
A bug in pytorch implementation of Auxiliary Classifier GAN 

### ISSUE THREAD

**Issue by EthanZhu90** [wiseodd/generative-models#20]
D_aux = torch.nn.Sequential(
    torch.nn.Linear(h_dim, y_dim),
    torch.nn.Softmax()
)
should be changed to :
D_aux = torch.nn.Sequential(
    torch.nn.Linear(h_dim, y_dim)
)

Because D_aux is the input of nn.cross_entropy(C_real/D_aux, y_true), where D_aux is output of linear() instead of softmax().

**Comment by LukasMosser** [comment]
Can you make a pull request?

**Comment by wiseodd** [comment]
@EthanZhu90 sorry for long delay.
Indeed you are correct. I've fixed this.
Thanks!
